content/news/buzz/july-2025-status-update/contents.lr

```diff
@@ -0,0 +1,49 @@
+title: July 2025 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+In July, BeeWare hit some more major milestones in our work on mobile packaging - but we also found time to make some improvements to other parts of BeeWare.
+
+## What we've done
+
+- We attended [EuroPython](https://ep2025.europython.eu), where we gave a tutorial and a talk, attended the WebAssembly summit, and ran a sprint - at which we gave out 25 BeeWare challenge coins to first-time contributors.
+- [cibuildwheel 3.1](https://pypi.org/project/cibuildwheel/3.1.0/) has been released - which includes support for building Android wheels! As a result, we've been able to submit a pull request for [lru-dict](https://github.com/amitdev/lru-dict/pull/74); we're expecting to submit Android support PRs for other projects in the coming months.
+- We started the process of [adding Android builds to CPython's CI](https://github.com/python/cpython/pull/137186). This also means we're in a position to [generate Android release artefacts as part of the release process](https://github.com/python/release-tools/pull/265) - possibly as soon as Python 3.14.
+- The [Emscripten buildbot is now passing all tests](https://buildbot.python.org/#/builders/1810)! This formally [restores Emscripten to the Tier-3 support level](https://peps.python.org/pep-0011/#tier-3).
+- [Pillow 11.3.0](https://pypi.org/project/pillow/11.3.0/#files) has been released, including official iOS wheels. 
+- We submited a pull request to add [iOS support for NumPy](https://github.com/numpy/numpy/pull/28759). This PR is waiting on a release of `meson-python` before it can be submitted for review, but it is passing the NumPy test suite.
+- [Briefcase has been updated to use WiX 5.0](https://github.com/beeware/briefcase/pull/2355). This enabled us to resolve a number of issues related to MSI support, including resolving an issue with [long file names in packaged content](https://github.com/beeware/briefcase/pull/2390).
+- We added support for [port forwarding when running Android apps](https://github.com/beeware/briefcase/pull/2369). This will be used as part of adding debugger support to Briefcase.
+- We added a new [topic guide on publishing apps to the macOS App Store](https://github.com/beeware/briefcase/pull/2360).
+- We improved a number of error messages generated by Briefcase, providing more guidance on [how to run an iOS app on device](https://github.com/beeware/briefcase/pull/2410), [problems that can be caused by virus scanners](https://github.com/beeware/briefcase/pull/2400), and how to resolve [inconsistent file content in macOS wheels](https://github.com/beeware/briefcase/pull/2397).
```

Comment:
Unless "on device" is a technical phrase, this should probably be updated.

```suggestion
- We improved a number of error messages generated by Briefcase, providing more guidance on [how to run an iOS app on a device](https://github.com/beeware/briefcase/pull/2410), [problems that can be caused by virus scanners](https://github.com/beeware/briefcase/pull/2400), and how to resolve [inconsistent file content in macOS wheels](https://github.com/beeware/briefcase/pull/2397).
```

Reply:
It is... ish. "On device" and "on a device" would both be correct; "on a device" is probably less confusing to a non-technical audience.